### ISSUE TITLE
Implement Glossification in WYISWYG

### ISSUE THREAD

**Issue by VictoriaSunNIH** [NCIOCPL/cgov-digital-platform#917]
We need to implement glossification in the WYSIWYG - including implementing how it will connect to the glossification service to glossify the terms.

Example Cancer.gov pages with glossified terms:
https://www.cancer.gov/about-cancer/understanding/what-is-cancer
https://www.cancer.gov/about-cancer/treatment

**Comment by bryanpizzillo** [comment]
Current JS code https://github.com/NCIOCPL/wcms-percussion-extensions/tree/master/Rhythmyx/rx_resources/tinymce/js/tinymce/plugins/glossifier

**Comment by bkline** [comment]
I have added a wiki page to describe a proposed API for the glossifier at https://github.com/NCIOCPL/cgov-digital-platform/wiki/Glossifier-service-API. It doesn't look like the wiki lets us attach files, so to supplement the shorter sample `JSON` given on the wiki page I have attached a more substantial example as
[glossifier-sample.zip](https://github.com/NCIOCPL/cgov-digital-platform/files/2945833/glossifier-sample.zip) (GitHub won't let me attach the `.json` files directly, but it will let me attach a `.zip` file containing those files).

Let me know if this proposed API will work for you, @BrendanBeltzNIH.
